src/Malt.jl

```diff
@@ -632,7 +632,31 @@ function interrupt(w::InProcessWorker)
     nothing
 end
 
+"""
+    Malt.requestgc(w::Worker)
 
+Request a garbage collection on the worker `w`. This is a non-blocking call (on the worker).
```

Comment:
You could add in these docs: "For a blocking GC, you can use `Malt.remote_call_wait(GC.gc, w)`."

I think the sentence `This is a non-blocking call (on the worker).` could be expanded a bit more, and be more technical about timing and throttling